docs/android/manifest.md

```diff
@@ -0,0 +1,246 @@
+---
+title: "Android app manifest"
+description: Learn about the Android app manifest, AndroidManifest.xml, that describes essential information about your app to build tools, the Android OS, and Google Play.
+ms.date: 03/07/2023
+---
+
+# Android app manifest
+
+Every .NET Multi-platform App UI (.NET MAUI) app on Android has an *AndroidManifest.xml* file, located in the *Platforms\\Android* folder, that describes essential information about your app to build tools, the Android operating system, and Google Play.
+
+The manifest file for your .NET MAUI Android app is generated as part of the .NET MAUI build process on Android. This build process takes the XML in the *Platforms\\Android\\AndroidManifest.xml* file, and merges it with any XML that's generated from specific attributes on your classes. The resulting manifest file can be found in the *obj* folder. For example, it can be found at *obj\\Debug\\net7.0-android\\AndroidManifest.xml* for debug builds on .NET 7.
+
+<!-- TODO: Uncomment when VS 17.6 P2 is released
+> [!NOTE]
+> Visual Studio 17.6 Preview 2 and higher includes a new editor that simplifies the process of specifying app details, the target Android version, and required permissions in an Android manifest file.
+-->
+
+## Generating the manifest
+
+All .NET MAUI app's have a `MainActivity` class that derives from <xref:Android.App.Activity>, via the `MauiAppCompatActivity` class, and that has the <xref:Android.App.ActivityAttribute> applied to it. Some apps may include additional classes that derive from <xref:Android.App.Activity> and that have the <xref:Android.App.ActivityAttribute> applied.
+
+At build time, assemblies are scanned for non-`abstract` classes that derive from <xref:Android.App.Activity> and that have the <xref:Android.App.ActivityAttribute> applied. These classes and attributes are used to generate the app's manifest. For example, consider the following code:
+
+```csharp
+using Android.App;
+
+namespace MyMauiApp;
+
+public class MyActivity : Activity
+{
+}
+```
+
+This example results in nothing being generated in the manifest file. For an `<activity/>` element to be generated, you'd need to add the <xref:Android.App.ActivityAttribute>:
+
+```csharp
+using Android.App;
+
+namespace MyMauiApp;
+
+[Activity]
+public class MyActivity : Activity
+{
+}
+```
+
+This example causes the following XML fragment to be added to the manifest file:
+
+```xml
+<activity android:name="crc64bdb9c38958c20c7c.MainActivity" />
+```
+
+> [!NOTE]
+> <xref:Android.App.ActivityAttribute> has no effect on `abstract` types.
+
+## Activity name
```

Comment:
This is how you could get a readable Java name for your activity, it can be useful if another Android app needs to be able to open your app (like Facebook login, for example). Or if you simply have a script for launching your app & testing startup time, etc.